### ISSUE TITLE
Accessibility Evaluation of Text with Texture Background

### ISSUE THREAD

**Issue by anitaIsTyping** [w3c/wcag#4479]
Hi,

I would like to ask for your support in evaluating the accessibility of a large text where, instead of a solid color, there is a texture as the background (as shown in the attached example, **EXAMPLE 1**). 
I find it difficult to read and I’m concerned that such a design might be hard to interpret for some users, causing issues. However, I would appreciate your opinion to understand if, according to the WCAG guidelines, it meets the necessary requirements.

I have the same question regarding **EXAMPLE 2**, where a dark border surrounds the letters, and compared to the white background of the site, it exceeds the 3:1 contrast ratio (in this case, also 4.5:1), making the shape of the letters more visible.

Is it possible that, both in **EXAMPLE 1** and **EXAMPLE 2**, given that some of the texture colors do not exceed the 3:1 contrast ratio relative to the white background of the page (according to WCAG Contrast (Minimum) (Level AA)), the title might not be designed in a compliant way? Or, in the case of **EXAMPLE 2**, can we consider it compliant and readable?

Thank you very much for your support!

**_EXAMPLE 1:_**
<img width="1150" alt="Image" src="https://github.com/user-attachments/assets/025a5d2a-91d0-475d-86a7-bedc72dac541" />

**_EXAMPLE 2:_**
<img width="1184" alt="Image" src="https://github.com/user-attachments/assets/9ecf7b9c-87cc-4c8a-96c9-353de333709f" />

**Comment by a11ydoer** [comment]
@anitaIsTyping  Thanks for filing this issue. Unfortunately, this topic is beyond the scope of the [ARIA Authoring Practice Guide. ](https://www.w3.org/WAI/ARIA/apg/about/introduction/) I am transferring this issue to the WCAG group if there is any help or guidance on this topic.

**Comment by a11ydoer** [comment]
This issue was beyond the ARIA APG Scope, and it was transferred to the WCAG group.

**Comment by TestPartners** [comment]
I would say that example 1 is clearly non-conformant, but it raises the question of how much of the text needs to have insufficient contrast before we report a non-conformance. Arguably, the entire perimeter of the text must, but it seems harsh to report a non-conformance if only one pixel has insufficient contrast.

This is similar to SC 2.4.7: Focus Visible, which we already agree passes as long as one pixel changes on focus. For consistency, we should agree that every pixel of the perimeter of the text must have sufficient contrast. I don't see what else we can do, given that the test is objective and binary.

I would say that example 2 is clearly conformant because every pixel of the perimeter of the text has sufficient contrast.
